### PR TITLE
Easy peasy Performancy

### PR DESCRIPTION
**Pull request by rrahir**
## Description:

description of this task, what is implemented and why it is implemented that way.

Task: : [TASK_ID](https://www.odoo.com/web#id=TASK_ID&action=333&active_id=2328&model=project.task&view_type=form&cids=1&menu_id=4720)

## review checklist

- [ ] feature is organized in plugin, or UI components
- [ ] support of duplicate sheet (deep copy)
- [ ] in model/core: ranges are Range object, and can be adapted (adaptRanges)
- [ ] in model/UI: ranges are strings (to show the user)
- [ ] undo-able commands (uses this.history.update)
- [ ] multiuser-able commands (has inverse commands and transformations where needed)
- [ ] new/updated/removed commands are documented
- [ ] exportable in excel
- [ ] translations (\_t("qmsdf %s", abc))
- [ ] unit tested
- [ ] clean commented code
- [ ] track breaking changes
- [ ] doc is rebuild (npm run doc)
- [ ] status is correct in Odoo

### CODE REVIEW

File: src/plugins/ui/selection.ts [hunk 1]
```diff
@@ -327,6 +327,9 @@ export class GridSelectionPlugin extends UIPlugin {
         this.selectedFigureId = null;
         break;
     }
+  }
+
+  finalize(): void {
```
Comment (hokolomopo) [inline src/plugins/ui/selection.ts:332]: Typo in commit name :)

**Review by hokolomopo — COMMENTED** [review]
Easy peasy lemon squeezy

File: src/plugins/ui/sheetview.ts [hunk 2]
```diff
@@ -156,8 +156,12 @@ export class SheetViewPlugin extends UIPlugin {
   }
 
   handle(cmd: Command) {
-    this.cleanViewports();
-
+    switch (cmd.type) {
+      case "UNDO":
+      case "REDO":
```
Comment (LucasLefevre) [inline src/plugins/ui/sheetview.ts:161]: undo/redo are already handled (differently) below ? is it normal ?

Comment (rrahir) [reply src/plugins/ui/sheetview.ts:161]: the cleanViewport is a pre-step, unrelated to the actual action we want to do with undo/redo but I could add 'cleanViewports' in  the al-ready existing switch case. It seemed clearer in my head to isolate this logic

File: src/plugins/ui/sheetview.ts [hunk 3]
```diff
@@ -157,8 +157,6 @@ export class SheetViewPlugin extends UIPlugin {
 
   handle(cmd: Command) {
     switch (cmd.type) {
-      case "UNDO":
-      case "REDO":
       case "DELETE_SHEET":
```
Comment (LucasLefevre) [inline src/plugins/ui/sheetview.ts:160]: then this case should be part of the existing switch/case

**Review by LucasLefevre — APPROVED** [review]

File: src/plugins/ui/sheetview.ts (hunk 3 above)
Comment (rrahir) [reply src/plugins/ui/sheetview.ts:160]: I did a fixup addressing it :) It's merged together

**Review by LucasLefevre — COMMENTED** [review]
robodoo r+ rebase-ff

**Pull request closed without merging**